### ISSUE TITLE
Alert syntax causing page truncation

### ISSUE THREAD

**Issue by cameronelliott** [h-enk/getdoks.org#125]
## TL;DR

**I figured out how to fix this. (I know very little about Hugo)
See last comment in chain for resolution.**
** note I only see this issue on the starter-theme, not the child theme, not sure what is going on **

## Description

Clean build of `github.com/h-enk/doks.git` has truncated content AFTER the first alert at this url: `http://localhost:1313/docs/prologue/introduction/`

### Steps to reproduce

```
cd
git clone https://github.com/h-enk/doks.git foo
cd foo
npm install
npm run start
open http://localhost:1313/docs/prologue/introduction/     # different terminal window
# scroll down to alert and see truncated content
```

### Expected result

Page should include multiple alerts and `Quick Start` heading.

### Actual result

Page includes single alerts and does not include `Quick Start` heading.

Basically, page should look like: [https://getdoks.org/docs/prologue/introduction/](https://getdoks.org/docs/prologue/introduction/)
But, the  page does not, the page appears truncated.

What happened.

### Environment

c@camerons-mini ~/foo (master)> npm run check

> @hyas/doks@0.2.3 precheck
> npm version

{
  '@hyas/doks': '0.2.3',
  npm: '7.5.3',
  node: '15.10.0',
  v8: '8.6.395.17-node.25',
  uv: '1.41.0',
  zlib: '1.2.11',
  brotli: '1.0.9',
  ares: '1.17.1',
  modules: '88',
  nghttp2: '1.42.0',
  napi: '7',
  llhttp: '2.1.3',
  openssl: '1.1.1j',
  cldr: '38.1',
  icu: '68.2',
  tz: '2020d',
  unicode: '13.0'
}

> @hyas/doks@0.2.3 check
> hugo version

hugo v0.83.1-5AFE0A57+extended darwin/amd64 BuildDate=2021-05-02T14:38:05Z VendorInfo=gohugoio
c@camerons-mini ~/foo (master)>

**Comment by h-enk** [comment]
See also: https://github.com/h-enk/doks/pull/235. Added [Alerts](https://getdoks.org/docs/recipes/alerts/) page.

**Comment by h-enk** [comment]
683ddd0b4702780feade1cbe7a2001da86742052
